modules/graphql_content/src/Plugin/GraphQL/Fields/RawValueField.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Drupal\graphql_content\Plugin\GraphQL\Fields;
+
+use Drupal\Core\Entity\FieldableEntityInterface;
+use Drupal\graphql_core\GraphQL\FieldPluginBase;
+use Youshido\GraphQL\Execution\ResolveInfo;
+
+/**
+ * Generic field plugin for rendering entity fields to string values.
+ *
+ * @GraphQLField(
+ *   id = "raw_field",
+ *   nullable = true,
+ *   weight = -1,
```

Comment:
... and that made the tests green. Strangely, I wasn't able to reproduce the failure locally so it was really puzzling. You rock! :1st_place_medal: